docs/howtoguides/trusty_legacy_support.rst

```diff
@@ -0,0 +1,105 @@
+How to enable esm-infra-legacy on Trusty
+*****************************************
+
+The ``esm-infra-legacy`` service can be enabled to grant additional two years of 
+extend security coverage. This service is only available on Trusty, since the release has reached
+its end of life support for esm-infra.
```

Comment:
```suggestion
the end of its support period for ``esm-infra``. With ``esm-infra-legacy``, the
support is extended until May 2026.
```

Reply:
(I assume it's May 2026, but please feel to correct me, or to be more specific if we know the actual date already)